packages/bot-utils/src/utils.ts

```diff
@@ -263,11 +252,32 @@ traceroute jsdelivr.com from Paris --port 453
       --share             Prints a link at the end the results, allowing to vizualize the results online (default false)`,
 };
 
-export const help: Help = {
-	help: generalHelpTexts,
-	dns: dnsHelpTexts,
-	http: httpHelpTexts,
-	mtr: mtrHelpTexts,
-	ping: pingHelpTexts,
-	traceroute: tracerouteHelpTexts,
+export const authHelpTexts = {
+	preamble: `Authenticate with the Globalping API for higher measurements limits.
+
+Available Commands:
+  login       Log in to your Globalping account
+  logout      Log out from your Globalping account
+  status      Check the current authentication status`,
+	usage: (rootCommand: string) => `${rootCommand} auth [command]`,
+	flags: '  -h, --help   Help for auth',
+};
+
+export const authLoginHelpTexts = {
+	preamble: 'Log in to your Globalping account for higher measurements limits.',
+	usage: (rootCommand: string) => `${rootCommand} auth login  [flags]`,
+	flags: `  -h, --help         Help for login
+      --with-token   Authenticate with a token instead of the default browser-based flow`,
```

Comment:
No --with-token in this app. I don't want people sending tokens in public channels.